sdk/core/abort-controller/src/aborter.ts

```diff
@@ -1,4 +1,4 @@
-/// <reference lib="es5" />
+// <reference lib="es5" />
```

Comment:
Please remove this reference comment entirely instead of turning it into a normal one

Reply:
@xirzec Removed that line and updated the PR. Please have a look